scripts/src/shika.js

```diff
@@ -0,0 +1,21 @@
+function shika_send() {
+    const url = document.currentScript.getAttribute("url");
```

Comment:
We should probably use `data-*` attributes, so `.getAttribute("url")` -> `.dataset.url`